### PR TITLE
Lesson5_Task1

### PR DESCRIPTION
**Pull request by stalgatuly**
1) Check the code with lint
2) Unit tests for shape methods (Area, Perimeter) that have 100% coverage.

### CODE REVIEW

File: lesson_05/shapes/Rectangle_test.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+)
+
+func TestRectangle_Area(t *testing.T) {
+	tests := []struct {
+		name    string
+		r       Rectangle
+		want    float64
+		wantErr bool
+	}{
+		// TODO: Add test cases.
```
Comment (denislituev) [inline lesson_05/shapes/Rectangle_test.go]: Don't forget to remove TODO labels if method is completed.

**Review by denislituev — APPROVED** [review]

File: lesson_05/shapes/chtml.out [hunk 2]
```diff
@@ -0,0 +1,23 @@
+mode: set
```
Comment (nikonok) [inline lesson_05/shapes/chtml.out]: We don't really need that file at request

File: lesson_05/shapes/Circle_test.go [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+	}{
+		// TODO: Add test cases.
+		{
+			name: "regular",
+			c:    Circle{2},
+			want: "Circle: radius 2.00",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
```
Comment (nikonok) [inline lesson_05/shapes/Circle_test.go:101]: I guess, we don't need a `for` if we want to check only one statement. But mb I think wrong=)

**Review by nikonok — APPROVED** [review]
See my comments, and do smth with them=)
After it you can merge request

**Pull request merged**